ui/library/less/styles.less

```diff
@@ -20,21 +14,62 @@
 
 .reader__page-overlay__bounding-box {
   position: absolute;
-  z-index: @z-index-bbox-overlay;
+  z-index: @z-index-bounding-box-overlay;
 }
 
 .reader__page-overlay__bounding-box-highlighted {
-  background-color: @pdflib-highlight-background;
-  opacity: .2;
+  background-color: @initial-blue;
+  mix-blend-mode: screen;
+
+  &:hover {
```

Comment:
@carolinepaulic gotta separate it out otherwise it will combine the style which will not work

Reply:
@huytr1995 IM me? seems odd